VultisigApp/VultisigApp/Utils/URL.swift

```diff
@@ -12,4 +12,6 @@ class StaticURL {
     static let XVultisigURL = URL(string: "https://twitter.com/vultisig")!
     static let AppStoreVultisigURL = URL(string: "https://www.apple.com/")!
     static let DiscordVultisigURL = URL(string: "https://discord.gg/ngvW8tRRfB")!
+    static let PrivacyPolicyURL = URL(string: "https://docs.google.com/document/d/1gxA5OcCFGupWDyF3VCyKwwojA99CVngH9fLER8irRvI/edit#heading=h.1de5j5yfesa8")!
```

Comment:
We need to wait for `vultisit.com` update the link to T&C , and Privacy Policy , then we can use the link

Reply:
Ah ok, will draft PR until then.

Reply:
@Rockindash see:
TOS: https://vultisig.com/termofservice
privacy: https://vultisig.com/privacy